src/liveEditor/utils/handleIndividualFields.ts

```diff
@@ -153,44 +156,45 @@ export async function handleIndividualFields(
 
 export function cleanIndividualFieldResidual(elements: {
     overlayWrapper: HTMLDivElement;
-    previousSelectedEditableDOM: Element;
-    visualEditorWrapper: HTMLDivElement | null;
+    visualEditorContainer: HTMLDivElement | null;
     focusedToolbar: HTMLDivElement | null;
 }): void {
-    const {
-        overlayWrapper,
-        previousSelectedEditableDOM,
-        visualEditorWrapper,
-        focusedToolbar,
-    } = elements;
+    const { overlayWrapper, visualEditorContainer, focusedToolbar } = elements;
 
     removeAddInstanceButtons({
         eventTarget: null,
-        visualEditorWrapper: visualEditorWrapper,
+        visualEditorContainer: visualEditorContainer,
         overlayWrapper: overlayWrapper,
     });
 
-    removeReplaceAssetButton(visualEditorWrapper);
+    removeReplaceAssetButton(visualEditorContainer);
 
-    const pseudoEditableElement = visualEditorWrapper?.querySelector(
+    const pseudoEditableElement = visualEditorContainer?.querySelector(
         ".visual-editor__pseudo-editable-element"
     );
 
-    previousSelectedEditableDOM.removeAttribute(
+    VisualEditorGlobalState.value.previousSelectedEditableDOM!.removeAttribute(
         LIVE_EDITOR_FIELD_TYPE_ATTRIBUTE_KEY
     );
-    previousSelectedEditableDOM.removeAttribute("contenteditable");
-    previousSelectedEditableDOM.removeEventListener("input", handleFieldInput);
-    previousSelectedEditableDOM.removeEventListener(
+    VisualEditorGlobalState.value.previousSelectedEditableDOM!.removeAttribute(
+        "contenteditable"
+    );
+    VisualEditorGlobalState.value.previousSelectedEditableDOM!.removeEventListener(
+        "input",
+        handleFieldInput
+    );
+    VisualEditorGlobalState.value.previousSelectedEditableDOM!.removeEventListener(
```

Comment:
Avoid using global state. This can cause problem with overrides during multiple instances

Reply:
I'll try to move it into the `VisualEditor` class.